sdk/src/wallet/account/operations/transaction/high_level/minting/create_native_token.rs

```diff
@@ -91,8 +91,8 @@ where
 {
     /// Creates a new foundry output with minted native tokens.
     ///
-    /// Calls [Account::send_outputs()](crate::wallet::Account::send_outputs) internally, the options may define the
-    /// remainder value strategy or custom inputs. Note that addresses need to be bech32-encoded.
+    /// Calls [Account::prepare_transaction()](crate::wallet::Account::prepare_transaction) internally, the options may
+    /// define the remainder value strategy or custom inputs. Note that addresses need to be bech32-encoded.
```

Comment:
cannot select the lines below, but the the params aren't correct either

Reply:
What's wrong with the params?

Reply:
nothing